datasource/src/main/scala/quasar/blobstore/azure/AzureBlobstore.scala

```diff
@@ -122,4 +112,13 @@ class AzureBlobstore[F[_]: Concurrent](
     val ns = normalize(s)
     ns.substring(ns.lastIndexOf('/') + 1)
   }
+
+  private def toByteStream(r: DownloadResponse): F[Stream[F, Byte]] =
+    F.delay {
+      for {
+        buf <- rx.flowableToStream(r.body(new ReliableDownloadOptions), maxQueueSize.value)
+        b <- Stream.chunk(Chunk.ByteBuffer(buf))
```

Comment:
:golf: `Chunk.byteBuffer(buf)`

Reply:
Yeah, its not a big deal, just a bit better encapsulated than referring to the class directly.